app/views/admin/admins/_form.html.haml

```diff
@@ -3,7 +3,7 @@
   = f.input :given_names, input_html: {maxlength: User::MAX_NAME_LENGTH}
   = f.input :family_name, input_html: {maxlength: User::MAX_NAME_LENGTH}
   = f.input :email
```

Comment:
Could be that the browser is smart enough to find a "new-password" field and will leave that field alone. Similarly, it could autocomplete the current password based on the provided username: https://html.spec.whatwg.org/multipage/forms.html#attr-fe-autocomplete-current-password